pkg/controller/eventing/inmemory/channel/reconcile_test.go

```diff
@@ -519,6 +519,34 @@ func TestReconcile(t *testing.T) {
 				events[channelReconciled],
 			},
 		},
+		{
+			Name: "Channel reconcile successful - Async channel",
+			InitialState: []runtime.Object{
+				makeChannel("in-memory"),
+			},
+			Mocks: controllertesting.Mocks{},
+			WantPresent: []runtime.Object{
+				makeVirtualService(),
+			},
+			WantEvent: []corev1.Event{
+				events[channelReconciled],
+			},
+		},
+		{
+			Name: "Channel reconcile successful - Non Async channel",
+			// VirtualService should have channel provisioner name
+			// overwritten to "in-memory"
+			InitialState: []runtime.Object{
+				makeChannel("in-memory-channel"),
+			},
+			Mocks: controllertesting.Mocks{},
+			WantPresent: []runtime.Object{
+				makeVirtualService(),
```

Comment:
Also `makeK8sService("in-memory")`

Reply:
done

Reply:
Sorry, I think this should have been `makeK8sService("in-memory-channel")`. Sorry for the confusion.